### ISSUE TITLE
Is possible to group all workshops in a single npm repository or a way to search workshop?

### ISSUE THREAD

**Issue by ErickWendel** [workshopper/org#47]
# Thank you for posting an issue!

I'm thinking to avoid developers use `npm i -g` for each workshop. We could use something like the `yo` where users could search for a npm package as shown below

![image](https://user-images.githubusercontent.com/8060102/58591726-4d1ef080-823d-11e9-8db5-f73183ec78d3.png)

**Comment by martinheidegger** [comment]
Yes, it is possible to do that, but it would take some work. If there was an effort in that direction I would welcome it.
